types/wordpress__notices/index.d.ts

```diff
@@ -13,12 +13,46 @@ declare module '@wordpress/data' {
 }
 
 export type Status = 'error' | 'info' | 'success' | 'warning';
+export type WPNoticeType = 'snackbar' | 'default';
 
 export interface Notice {
+    /**
+     * Unique identifier of notice.
+     */
     id: string;
+    /**
+     * Status of notice, one of `success`, `info`, `error`, or `warning`. Defaults to `info`.
+     */
     status: Status;
+    /**
+     * Notice message.
+     */
     content: string;
+    /**
+     * Audibly announced message text used by assistive technologies.
+     */
+    spokenMessage: string;
+    /**
+     * Notice message as raw HTML. Intended to serve primarily for compatibility of server-rendered notices, and SHOULD
+     * NOT be used for notices. It is subject to removal without notice.
+     */
+    __unstableHTML: string;
```

Comment:
this looks like private by convention. Is this expected to be available in TS?

Reply:
Thank you for the feedback!

I understand the concern, but in the test here https://github.com/WordPress/gutenberg/blob/%40wordpress/notices%403.5.0/packages/notices/src/store/test/actions.js#L90 it is used. On this basis I am not sure if removing it from the type definition is the correct thing to do if it is in fact a property that can be accessed?

If it was removed from the type definition then that code would be considered incorrectly typed wouldn't it?